frontend/src/containers/ResetPassword/ResetPasswordPage.js

```diff
@@ -1,23 +1,30 @@
 import React from 'react';
+
+import querystring from 'querystring';
 import LogInPageHeader from '../../components/LogInPageHeader/LogInPageHeader';
 import NewPassword from '../../components/NewPassword/NewPassword';
 import Footer from '../../components/Footer/Footer';
 import './ResetPasswordPage.css';
 
-const ResetPasswordPage = () => (
-  <div className="resetPasswordPage container-fluid">
-    <div className="row">
-      <LogInPageHeader />
-    </div>
-    <div className="passwordBackgroundImage">
-      <div className="newPasswordCardWrapper">
-        <NewPassword />
-        <div className="loginFooter row">
-          <Footer />
+const ResetPasswordPage = (props) => {
+  const { location } = props;
+  const parsed = querystring.parse(location.search.slice(1));
+  const token = parsed.q;
```

Comment:
I'd probably use `token` as the query param, though we'll need to see what rails wants to do.